retis/src/process/cli/pcap.rs

```diff
@@ -47,12 +51,14 @@ struct EventParserStats {
 /// Events parser: handles the logic to convert our events to the PCAP format
 /// that is represented by the internal writer.
 struct EventParser {
-    /// Known network interfaces and their PCAP id: netns|ifindex -> pcap id.
-    ifaces: HashMap<u64, u32>,
+    /// Known network interfaces and their PCAP ids: key -> pcap id.
+    ifaces: HashMap<String, u32>,
```

Comment:
It's probably time to rework the comment and rename this? Or at least explain briefly "interface" does not mean "interface" :)